### PR TITLE
chore(new_exp): Add docs for ebs loss and ec2 terminate experiments

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

**What this PR does / why we need it**:

- This PR is for the addition of docs for new experiments which are: 
  - [X] EBS Loss
  - [X] EC2 Terminate

- It also contains embedmd run for pod-cpu-hog and pod-memory-hog changes.
- Addition of the experiment in sidebar.json also modified the name of chaos toolkit ec2 terminate experiment.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [x] Signed the commit for [DCO](https://github.com/litmuschaos/litmus-docs/blob/master/CONTRIBUTING.md#sign-your-work) check to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Review by ksatchit — DISMISSED** [review]

File: docs/ebs-loss.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  </tr>
+  <tr> 
+    <td> TOTAL_CHAOS_DURATION </td>
+    <td> The time duration for chaos insertion (sec) </td>
+    <td> Optional </td>
+    <td> Defaults to 60s </td>
+  </tr>
+  <tr>
+    <td> REGION </td>
+    <td> The region name of the target instace</td>
```
Comment (ispeakc0de) [inline docs/ebs-loss.md]: instace -> instance

File: docs/ebs-loss.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  
+- Monitor the attachment status for ebs volume from AWS CLI.
+
+  `aws ec2 describe-volumes --volume-ids <vol-id>`
+
+-  You can also use aws console to keep a watch over ebs attachment status.   
+
+### Check Chaos Experiment Result
+
+- Check whether the application is resilient to the container kill, once the experiment (job) is completed. The ChaosResult resource name is derived like this: `<ChaosEngine-Name>-<ChaosExperiment-Name>`.
```
Comment (ispeakc0de) [inline docs/ebs-loss.md]: container kill? Can we change it to ebs-loss?

File: docs/ec2-terminate.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    <td> Termination of an EC2 instance for a certain chaos duration</td>
+    <td> EKS </td>
+  </tr>
+</table>
+
+## Prerequisites
+
+- Ensure that Kubernetes Version > 1.13
+- Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
+- Ensure that the `ec2-terminate` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace If not, install from [here](https://hub.litmuschaos.io/api/chaos/1.9.1?file=charts/generic/ec2-terminate/experiment.yaml)
```
Comment (ispeakc0de) [inline docs/ec2-terminate.md]: Can you verify the hub-link? it should be kube-aws

File: docs/ebs-loss.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+    <td> EBS volume loss against specified application </td>
+    <td> EKS </td>
+  </tr>
+</table>
+
+## Prerequisites
+
+- Ensure that Kubernetes Version > 1.13
+- Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
+- Ensure that the `ebs-loss` experiment resource is available in the cluster by executing `kubectl get chaosexperiments` in the desired namespace If not, install from [here](https://hub.litmuschaos.io/api/chaos/1.9.1?file=charts/generic/ebs-loss/experiment.yaml)
```
Comment (ispeakc0de) [inline docs/ebs-loss.md]: Can you verify the hub-link? it should be kube-aws

File: docs/ec2-terminate.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+  
+- Monitor the ec2 state from AWS CLI.
+
+  `aws ec2 describe-instance-status --instance-ids <instance-id>`
+
+-  You can also use aws console to keep a watch over the instance state.   
+
+### Check Chaos Experiment Result
+
+- Check whether the application is resilient to the container kill, once the experiment (job) is completed. The ChaosResult resource name is derived like this: `<ChaosEngine-Name>-<ChaosExperiment-Name>`.
```
Comment (ispeakc0de) [inline docs/ec2-terminate.md]: container kill -> ec2-terminate?

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**